packages/world/tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "strict": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "noErrorTruncation": true
```

Comment:
this makes increases the length of types before being truncated, which is useful for development